### ISSUE TITLE
Image Preview not working

### ISSUE THREAD

**Issue by KnownAsDon** [ranger/ranger#1089]
I am using manjaro i3 and ranger was preinstalled and configured. I have wanted to use ranger for so long and now that im using i love it but as u can see in the title image preview doesnt work. I have enabled it and tried w3m too( actually it was preconfigured) but when i try to preview and image it appears like [this](http://ibb.co/jt95yH). If you could help me I'd greatly appreciate it.

**Comment by nightsdark** [comment]
Found this on manjaro forum you could try: 
2 possible solutions in this link + 
https://forum.manjaro.org/t/ranger-image-previews/35393

----------------------------------------------------------------------------------------------------------------------------------------
If that worked great and if not next I would go https://github.com/ranger/ranger/wiki/Image-Previews
see if there's any solution for you there.

**Comment by KnownAsDon** [comment]
Yup, removing scope.sh and regenerating it fixed the problem, thanks man <3.

**Comment by toonn** [comment]
@KnownAsDon easier solution would be to remove `scope.sh` and the line `set preview_script ~/.config/ranger/scope.sh` from `rc.conf` That way you'll never end up with an old one again.
Only reason to have a `scope.sh` is if you customize it, same applies to `commands.py`, `rifle.conf` and `rc.conf`. In this case you'll need `rc.conf` because otherwise you can't remove that line.
